Cargo.toml

```diff
@@ -7,9 +7,11 @@ authors = ["The Rust-gnome Project Developers"]
 libc = "0.1"
 glib = "^0"
 gdk = "^0"
-gtk = "^0"
 cairo-rs = "^0"
 
+[dependencies.gtk]
+features = ["gtk_3_10", "cairo_1_12"]
```

Comment:
Alternatively, we could add the [`add_from_file`](https://developer.gnome.org/gtk3/stable/GtkBuilder.html#gtk-builder-add-from-file) method and avoid depending on 3.10 altogether.